### DISCUSSION TITLE
Filament front - meta tag on login page

### DISCUSSION THREAD

**Discussion by Grazulex** [filamentphp/filament#3892] [Help] [answered]
Hello,

For one of my projet, I used filament as root url (no "admin" url) but I need now to add meta tag on the login/registery pages. (I use for that the Breezy plugin)
I have find the "pushMeta" method but I don't know how and where I can use this for the login page.

Can somebody help me ?

Thanks.

**Comment by Z3d0X** [comment, marked as answer]
You would place it in the `boot()` method of a ServiceProvider (eg: `AppServiceProvider`)
```php
// app/Providers/AppServiceProvider.php
// ....
public function boot()
{
    Filament::pushMeta([
        new HtmlString('<meta name="description" content="New Filament site">'),
    ]);
}
```

**Reply by Grazulex** [reply]
Many thanks
